### DISCUSSION TITLE
is this compatible with terraria 1.4.1.2?

### DISCUSSION THREAD

**Discussion by Ghoose1** [popstarfreas/Dimensions#79] [General]
essentially just the title. also, if there are any videos or guides on how to install this and set it up other than the readme they would also be greatly appreciated.

**Comment by popstarfreas** [comment]
Yes the current example config is for 1.4.1.2. It also requires you to use tShock instead of a vanilla server since it's not currently compatible with vanilla servers (https://github.com/popstarfreas/Dimensions/pull/58)

Whether the README is sufficient depends on your prior experience.
For step 1 you just download and install nodejs. On windows it's the same as every other installer you might have seen.
On step 2 given you're using tShock you download the dll file and put it in the ServerPlugins/ folder.
Step 3 depends on if you've ever used a command prompt or terminal before.

If you explain what parts of the README steps 1-3 you're having trouble with I can provide more specific instructions depending on what you do or do not know.
